geotrek/common/management/commands/sync_rando.py

```diff
@@ -502,16 +505,13 @@ def handle(self, *args, **options):
             'ignore_errors': True,
             'tiles_dir': os.path.join(settings.VAR_DIR, 'tiles'),
         }
-        self.tmp_root = os.path.join(os.path.dirname(self.dst_root), 'tmp_sync_rando')
-        try:
-            os.mkdir(self.tmp_root)
-        except OSError as e:
-            if e.errno != 17:
-                raise
-            raise CommandError(
-                "The {}/ directory already exists. Please check no other sync_rando command is already running."
-                " If not, please delete this directory.".format(self.tmp_root)
-            )
+        if options['empty_tmp_folder']:
```

Comment:
```suggestion
        sync_rando_tmp_dir = os.path.join(settings.TMP_DIR, 'sync_rando')
        if options['empty_tmp_folder']:
```
os.path.join only once